lib/tasks/migrate_data.rake

```diff
@@ -99,4 +99,36 @@ namespace :tww do
       end
     end
   end
+
+  desc "migrate some test data from quickbooks"
```

Comment:
Please put this in a separate file unless there is some reason to keep it in this one.